src/components/CommentTeaser/CommentTeaser.js

```diff
@@ -71,7 +72,10 @@ const styles = {
   image: css({
     borderBottomWidth: 1,
     borderBottomStyle: 'solid',
-    width: '100%'
+    width: '100%',
+    [mUp]: {
+      marginTop: 20
```

Comment:
look weird / leads to double margin because `imageContainer` already has 20px for mUp
<img width="1566" alt="Screenshot 2021-02-03 at 00 13 08" src="https://user-images.githubusercontent.com/410211/106675060-add70280-65b4-11eb-9c59-80df0800786b.png">